packages/react-reconciler/src/SchedulerWithReactIntegration.new.js

```diff
@@ -51,80 +50,9 @@ if (enableSchedulerTracing) {
 
 export type SchedulerCallback = (isSync: boolean) => SchedulerCallback | null;
 
-type SchedulerCallbackOptions = {timeout?: number, ...};
-
-// Except for NoPriority, these correspond to Scheduler priorities. We use
-// ascending numbers so we can compare them like numbers. They start at 90 to
-// avoid clashing with Scheduler's priorities.
-export const ImmediatePriority: ReactPriorityLevel = 99;
-export const UserBlockingPriority: ReactPriorityLevel = 98;
```

Comment:
Goodbye, 99, 98, and friends. How many times have I seen you while debugging, trying to remember which level it is. Hope to never see you again! 🕺

Reply:
Hello to 0, 1, 2, 3, 4, 5!

Reply:
I'll take that

Reply:
How about 0, 1, 2, 4, 8, 16, 32, 64...? 😆

Reply:
If we stay below 65536 sure. I can't memorize the ones after.